packages/@aws-cdk/aws-cognito/test/test.user-pool.ts

```diff
@@ -68,6 +68,8 @@ export = {
     pool.addPostConfirmationTrigger(fn);
     pool.addPreAuthenticationTrigger(fn);
     pool.addPreSignUpTrigger(fn);
+    pool.addPreTokenGenerationTrigger(fn);
```

Comment:
Whoops, I just noticed that this test doesn't do what I **think** it was originally intended for.

Each of these `addXXX()` methods should take a different instances of `lambda.Function` and later the assertion should verify that the right lambda function ended up against the right trigger when the stack is resolved. 
Weirdly, it seems that they're all taking in the same lambda function, which defeats the purpose of this test.

@vas1468 - could we get this fixed up? It should be straight forward to have 10 instances of `lambda.Function` created and use them both in the `addXXX` method and the assertion (i.e. the `expect` clause)